how-to/customize-workspace/docs/how-to-customize-home.md

```diff
@@ -0,0 +1,112 @@
+
+For example, to only show results after a minimum of 3 characters have been entered, and to only search in the `title` field of the sources, we could use the following configuration.
+
+```json
+"homeProvider": {
+    ...
+    "queryMinLength": 3,
+    "queryAgainst": ["title"]
+}
+```
+
+There are some additional flags in the home configuration which can be used to turn on and off specific features.
+
+`enablePageIntegration` will enable listing, launching, deleting and sharing commands for interacting with Pages.
+`enableWorkspaceIntegration` will enable listing, launching, deleting and sharing commands for interacting with Workspaces and also the ability to save the current layout as a workspace using the `/w` command.
+
+Sharing for pages and workspaces will only be available if the `platformProvider.sharing` flag is enabled.
+
+```json
+"homeProvider": {
+    ...
+    "enablePageIntegration": true,
+    "enableWorkspaceIntegration": true
+}
+```
+
+## Sources and results
+
+The sources that provide the results for the home components fall into 2 categories, `apps`, `commands` and `search` results.
+
+- `Apps` are retrieved from the app sources configured for the platform, see [How To Define Apps](./how-to-define-apps.md)
+- `Commands` are fixed entries, there are some built-in like `/dock`, `/store`, `/notifications`, but these can be extended using custom commands, see [How to Add Integrations to Home](./how-to-add-integrations-to-home.md)
+- `Search` entries can be used to provide results from search and other endpoints, see [How to Add Integrations to Home](./how-to-add-integrations-to-home.md)
+
+Apps use a fixed template for display as a search result:
+
+![Apps Template](./assets/home-apps.png)
+
+Whereas commands and search entries can have custom templates with richer layouts:
+
+![Commands Template](./assets/home-commands.png)
+
+## Integration management
+
+The integrations for both `command` and `dynamic` results by default will start automatically if enabled in configuration. You can introduce finer grained control of this feature by enabling integration management. When this option is enabled you have the option to enable/disable integrations through home commands. The state of an integration is stored in local storage for the platform, so that when you restart the platform only the modules which are enabled will be initialized.
+
+You can use the `/integrations` command to determine which modules are available.
+
+```json
+"homeProvider": {
+    ...
+   "isManagementEnabled": true,
+   "command": "integrations",
+   "commandDescription": "Allows the management of integrations for this platform. You can decide whether enabled integrations should be included when a query is entered.",
+}
+```
+
+![Home Integrations](./assets/home-integrations.png)
+
```

Comment:
@obany could we expand a bit more here. I.e. if this is enabled what do you need to have on an integration module entry for it to support integration management (e.g. autostart false, enabled true etc). Can we also mention the endpoint entries that point to where the preference is saved/fetched and point to the define endpoints doc?

Reply:
Added